pom.xml

```diff
@@ -121,6 +121,12 @@
             <version>${lucene.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>elasticsearch</artifactId>
+            <version>1.11.3</version>
```

Comment:
BTW, 1.11.4 is the latest patch release of TestContainers for Java.

Reply:
got catch, this one was the latest when I started this work 1m ago 😄 .